content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+---
+title: API Server Flow Control
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+Controlling the behavior of the Kubernetes API server in an overload situation
+is a key task for cluster administrators. The{{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} has some controls
+available (i.e. the `--max-requests-inflight` and
+`--max-mutating-requests-inflight` command-line flags) to limit the amount of
+outstanding work that will be accepted, preventing a flood of inbound requests
+from overloading and potentially crashing the API server, but beyond that, the
+Priority and Fairness (or flow-control) API provides a configurable mechanism to
+ensure that in a high-load situation, forward progress will be made on the right
+requests. It also allows for a limited amount of queuing, so that in the event
```

Comment:
The wording "progress will be made on the right requests" makes it sound like there is a clear and simple binary judgement being made.  While it is true that ultimately there _is_ a binary decision (accept or reject), that is actually the result of a more complex process that I think of in terms of dividing up server resources among competing flows.

Reply:
I am all for sparing users from seeing implementation details, but this is a question of what this feature is all about.  Users of it need to clearly understand that it is about dividing server capacity among flows; in my opinion any attempt to make it look simpler will cause problems by giving users the wrong idea of what is going on.